### DISCUSSION TITLE
Issue - ModuleNotFoundError("No module named 'stability_sdk'")

### DISCUSSION THREAD

**Discussion by vvstephan** [carson-katri/dream-textures#385] [General]
Was hoping someone could help me with this, haha beating my head against the wall. Pretty new to blender, but not my first rodeo either, so I'm just looking for some friendly help! Sorry if this question has been asked.

I'm running the latest versions of python and git (if that matters) as well as the latest version of blender with the latest repo of dream textures.

**API key has been punched in, and I only get this error when I click on generate. Everything works perfect up to that point**.

Any help would be greatly appreciated!

**Comment by carson-katri** [comment]
Did you download a packaged build from the releases tab? If so, which one?

**Reply by vvstephan** [reply]
Thanks so much for your response! Yes, latest packaged build, v0.0.8 (Windows CUDA).

**Reply by carson-katri** [reply]
Can you try with the DreamStudio-only build?

**Reply by vvstephan** [reply]
Trying now, thanks!

**Reply by vvstephan** [reply]
That completely fixed it, thank you so much for your time man! Haha lifesaver. Keep making awesome stuff!

**Reply by carson-katri** [reply]
Awesome 👍
